jobs/regular/recalculate_scores.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Jobs
+  class RecalculateScores < ::Jobs::Base
+    def execute(opts = { since: "2014-8-26" })
```

Comment:
Why this date?

And if we need something, maybe default to 10.days.ago like the default?